cloudnetpy/output.py

```diff
@@ -289,13 +289,15 @@ def _get_dimensions(nc: netCDF4.Dataset, data: np.ndarray) -> tuple:
 
 
 def _get_identifier(short_id: str) -> str:
-    valid_ids = ("lwc", "iwc", "drizzle", "classification")
+    valid_ids = ("lwc", "iwc", "drizzle", "classification", "reff-Frisch")
```

Comment:
I agree with you to stick to the naming convention. reff originates (at least to my knowledge) from the Frisch 2002 paper. 'def' is much better, since the ice effective radius 'ief' (Hogan 2006) will follow soon.